src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -101,6 +113,33 @@ private void indexJarNames() throws ParserConfigurationException, IOException, S
         }
     }
 
+    public void openThanksPopUp(ActionEvent actionEvent) throws IOException{
+        try {
+            InputStream thanksFile = System.class.getResourceAsStream("/files/thanks_file");
+            String contentThanksFile = IOUtils.toString(thanksFile, "UTF-8");
+            showAboutPopUp(contentThanksFile);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Please use a logger or `alert(...);`
Because it's a user triggered action, an alert (that is shown to the user) would probably be best